src/app/new-auth/forgot-password/forgot-password.component.ts

```diff
@@ -0,0 +1,105 @@
+import { Component, OnInit } from '@angular/core';
+import { FormGroup } from '@angular/forms';
+import { RegisteredEmailValidator } from 'app/shared/validators/RegisteredEmailValidator';
+import { MatchValidator } from 'app/shared/validators/MatchValidator';
+import { AuthValidationService } from 'app/core/auth-validation.service';
+import { AuthService } from 'app/core/auth.service';
+import { debounce, takeUntil } from 'rxjs/operators';
+import { Subject, interval } from 'rxjs';
+import { animate, state, style, transition, trigger } from '@angular/animations';
+
+@Component({
+  selector: 'clark-forgot-password',
+  templateUrl: './forgot-password.component.html',
+  styleUrls: ['./forgot-password.component.scss'],
+  animations: [
+    trigger('toggleButton', [
+      state('disabled', style({
+          borderColor: 'grey',
+          backgroundColor: 'grey',
+          color: 'black',
+          opacity: 0.2
+      })),
+      state('enabled', style({
+          color: 'white',
+          opacity: 1
+      })),
+      transition('disabled <=> enabled', [
+        animate('0.2s')
+      ])
+    ]),
+    trigger('switchViews', [
+      transition(':enter', [
+        style({ opacity: 0 }),
+        animate('1000ms')
+      ])
+    ])
+  ]
+})
+export class ForgotPasswordComponent implements OnInit{
+  showError: Boolean = false;
+  errorMessage: String;
+  showDone: Boolean = false;
+  submitButton: String = 'disabled';
+
+  emails: FormGroup = new FormGroup({
+    'email': this.authValidationService.getInputFormControl('email'),
+    'confirmEmail': this.authValidationService.getInputFormControl('email')
```

Comment:
Since this is just a confirmation field it can just be required. Since the actual check for the email syntax is done in the email field